packages/astro/src/assets/internal.ts

```diff
@@ -36,7 +36,7 @@ export function isRemoteAllowed(
 		remotePatterns = [],
 	}: Partial<Pick<AstroConfig['image'], 'domains' | 'remotePatterns'>>
 ): boolean {
-	if (!isRemotePath(src)) return false;
+	if (!isRemotePath(src)) return true;
```

Comment:
@Princesseuh if you prefer, we can also ensure only images on public dir are allowed here by checking src as absolute path:

```suggestion
	if (!isRemotePath(src) && src.charAt(0) === '/') return true;
```